dashboard/app/handler.go

```diff
@@ -36,6 +36,7 @@ func handleContext(fn contextHandler) http.Handler {
 		if !throttleRequest(c, w, r) {
 			return
 		}
+		defer backpressureRobots(c, r)
 		if err := fn(c, w, r); err != nil {
```

Comment:
Does it work if we add a delay after writing complete response? HTTP encoding allows to understand when a client has received all bytes of the response, so perhaps they just don't wait after that?

Reply:
It works for curl and wget.
I'll check once more once we'll re-enable the throttling on appengine.